### PR TITLE
test: add a test helper for creating a spreadsheet from excel file

### PR DESCRIPTION
**Pull request by tomivirkki**
Multiple unit tests read an Excel file and initiate a `Spreadsheet` with the file used as the data source. This PR adds a shared helper for it.

### CODE REVIEW

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/test/java/com/vaadin/flow/component/spreadsheet/tests/TestHelper.java [hunk 1]
```diff
@@ -25,4 +29,32 @@ static void fireClientEvent(Spreadsheet spreadsheet, String eventName,
...
     }
 
+    /**
+     * Ceates a Spreadsheet component with the given Excel file as the data
+     * source.
+     *
+     * @param fileName
+     *            the name of the file. The file must be in the test_sheets
+     *            folder.
+     * @return the test sheet file
```
Comment (DiegoCardoso) [inline vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/test/java/com/vaadin/flow/component/spreadsheet/tests/TestHelper.java:39]: This doc seems a bit off.

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/test/java/com/vaadin/flow/component/spreadsheet/tests/TestHelper.java [hunk 2]
```diff
@@ -25,4 +29,32 @@ static void fireClientEvent(Spreadsheet spreadsheet, String eventName,
...
+    static Spreadsheet createSpreadsheet(String fileName) {
+        File testSheetFile = getTestSheetFile(fileName);
+        try {
+            return new Spreadsheet(testSheetFile);
+        } catch (IOException e) {
+            throw new RuntimeException("Could not create Spreadsheet", e);
+        }
+    }
+
+    static File getTestSheetFile(String name) {
```
Comment (DiegoCardoso) [inline vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/test/java/com/vaadin/flow/component/spreadsheet/tests/TestHelper.java:50]: Should we add a javadoc for this method?

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/test/java/com/vaadin/flow/component/spreadsheet/tests/TestHelper.java (hunk 1 above)
Comment (tomivirkki) [reply vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/test/java/com/vaadin/flow/component/spreadsheet/tests/TestHelper.java]: Done

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/test/java/com/vaadin/flow/component/spreadsheet/tests/TestHelper.java (hunk 2 above)
Comment (tomivirkki) [reply vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/test/java/com/vaadin/flow/component/spreadsheet/tests/TestHelper.java]: Done

**Review by DiegoCardoso — APPROVED** [review]

**Pull request merged**